nn_sdwan/MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+graft datadog_checks
```

Comment:
@yzhan289 It looks like this one may have been made before the full pyproject migration, would you be able to help bring this up to date? My understanding is that will be required before this can be merged.

Reply:
Communicated offline, but we can get this PR merged in now, and then @nikhilguptanetnologyio can submit a new PR with the pyproject migration.